Unit tests fail in develop branch

```
pytest tests/test_registry_management.py
```

results in a runtime exception

```
self = <dsgrid.registry.dimension_registry_manager.DimensionRegistryManager object at 0x7fa3182ba0d0>
config_file = PosixPath('/var/folders/pb/k6nlfrfj7n75gnl_6z880y50sf8tpd/T/test_us_data/dsgrid_project/dimensions.toml'), submitter = 'test_user'
log_message = 'registration message', force = False

    def register(self, config_file, submitter, log_message, force=False):
        lock_file_path = self.get_registry_lock_file(None)
>       with self.cloud_interface.make_lock_file(lock_file_path):
E       AttributeError: __enter__

dsgrid/registry/dimension_registry_manager.py:171: AttributeError
```

--- a/dsgrid/cloud/fake_storage_interface.py
+++ b/dsgrid/cloud/fake_storage_interface.py
@@ -19,10 +19,7 @@ class FakeStorageInterface(CloudStorageInterface):
 
     @contextmanager
     def make_lock_file(self, path):
-        try:
-            yield
-        finally:
-            pass
+        yield
 
     def read_lock_file(self, path):
         pass

